lib/org/openpsa/sales/handler/action.php

```diff
@@ -11,19 +11,9 @@
  *
  * @package org.openpsa.sales
  */
-class org_openpsa_sales_handler_sales_action extends midcom_baseclasses_components_handler
+class org_openpsa_sales_handler_action extends midcom_baseclasses_components_handler
 {
-    private $salesproject;
-
-    public function _on_initialize()
-    {
-        midcom::get()->auth->require_valid_user();
-        if (empty($_POST['id'])) {
-            throw new midcom_error('Incomplete POST data');
-        }
-        $this->salesproject = new org_openpsa_sales_salesproject_dba((int) $_POST['id']);
-        $this->salesproject->require_do('midgard:update');
-    }
+    private $_salesproject = null;
```

Comment:
Pls avoid underscores in the beginning of variable names for new code (the existing uses are still from PHP4 era)